Compiling v3.0.0 fails on x86

Tried to integrate the 3.0.0 release into vcpkg, but it fails due to:

```
2>C:\dev\libE57Format\src\CheckedFile.cpp(324,42): error C2220: the following warning is treated as an error
2>C:\dev\libE57Format\src\CheckedFile.cpp(324,42): warning C4244: 'argument': conversion from 'uint64_t' to 'size_t', possible loss of data
2>C:\dev\libE57Format\src\CheckedFile.cpp(334,45): warning C4244: 'argument': conversion from 'uint64_t' to 'size_t', possible loss of data
``` 

Sadly warnings as errors will be always enabled by default as `E57_BUILDING_SELF` will be set to `ON`.

--- a/src/CheckedFile.cpp
+++ b/src/CheckedFile.cpp
@@ -686,7 +686,7 @@ void CheckedFile::unlink()
 #endif
 }
 
-void CheckedFile::verifyChecksum( char *page_buffer, size_t page )
+void CheckedFile::verifyChecksum( char *page_buffer, uint64_t page )
 {
    const uint32_t check_sum = checksum( page_buffer, logicalPageSize );
    const uint32_t check_sum_in_page =

--- a/src/CheckedFile.h
+++ b/src/CheckedFile.h
@@ -89,7 +89,7 @@ namespace e57
       static inline uint64_t physicalToLogical( uint64_t physicalOffset );
 
    private:
-      void verifyChecksum( char *page_buffer, size_t page );
+      void verifyChecksum( char *page_buffer, uint64_t page );
 
       template <class FTYPE> CheckedFile &writeFloatingPoint( FTYPE value, int precision );
 

